doc/examples/example_asymm_crypto.md

```diff
@@ -394,3 +394,68 @@ mococrw::ECDSASignaturePublicKeyCtx verifyCtx = mococrw::ECDSASignaturePublicKey
 mococrw::ECDSASignaturePrivateKeyCtx signCtx = mococrw::ECDSASignaturePrivateKeyCtx(key, mococrw::openssl::DigestTypes::SHA512);
 ```
 
+## Signature Formats
+MoCOCrW supports two different formats for the signature:
+* `mococrw::ECDSASignatureFormat::ASN1_SEQUENCE_OF_INTS`:
+  Encoding of (r,s) as ASN.1 sequence of integers as specified in ANSI X9.62
+* `mococrw::ECDSASignatureFormat::IEEE1363`:
+  Encoding of (r,s) as raw big endian unsigned integers zero-padded to the key length as specified in IEEE 1363
+
+MoCOCrW offer the following overloads to specify the signature formats:
+* `mococrw::ECDSASignaturePublicKeyCtx(const mococrw::AsymmetricPublicKey&, mococrw::openssl::DigestTypes, mococrw::ECDSASignatureFormat)`
+* `mococrw::ECDSASignaturePrivateKeyCtx(const mococrw::AsymmetricPrivateKey&, mococrw::openssl::DigestTypes, mococrw::ECDSASignatureFormat)`
+
+# EdDSA
+
+Please note that EdDSA only implements signMessage/verifyMessage methods. That is, the hash to be signed can be specified manually.
```

Comment:
Did you mean: can *not* be specified manually, since that would require the signDigest/verifyDigest methods.